README.md

```diff
@@ -110,6 +110,10 @@ Short description.
 
   The UK's biggest green supplier. They provide members with 100% renewable electricity from solar, wind and hydro. Plus, the gas is 100% carbon neutral. 10% is green gas produced from renewable sources like food or farm waste.
 
+- [Carbon Engineering](https://carbonengineering.com/) | Canada | 2009
+
+  Using [Direct Air Capture](https://en.wikipedia.org/wiki/Direct_air_capture) technology, Carbon Engineering aims to take existing CO<sub>2</sub> directly out of the air, then using the gas to make carbon neutral synthetic fuels for planes, trucks, and ships. With support from Bill Gates and oil sands magnate Norman Murray Edwards, it's sure to see success in cleanly fueling global shipping infrastructure.
```

Comment:
I'm sure that most of the companies in this list are successful or will be, but there's no need to add that to the description ;)

```suggestion
  Using Direct Air Capture technology, Carbon Engineering aims to take existing CO<sub>2</sub> directly out of the air, then using the gas to make carbon neutral synthetic fuels for planes, trucks, and ships. Carbon Engineering is funded by several government and sustainability-focused agencies as well as by private investors, including Bill Gates and oil sands magnate Norman Murray Edwards.
```

Reply:
`then using the gas` > `then use the gas`

Reply:
Updated the suggestion, thank you for the correction @natstar93